### DISCUSSION TITLE
Change placeholder color when textarea is invalid (use-form)

### DISCUSSION THREAD

**Discussion by DKrawczyk** [mantinedev/mantine#4311] [Q&A]
Any ideas how can I change basic red color in textarea to default or just black?

When error occurs all textarea is changing od red, I don't want it.

I tried by:
```
 "& .mantine-Textarea-invalid": {
              color: "gray !important",
            },
```

but it still doesn't work

**Comment by rtivital** [comment]
https://codesandbox.io/s/jolly-voice-25jd8w?file=/src/App.tsx
